eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -274,10 +274,4 @@ stages:
         inputs:
           command: push
           packagesToPush: '$(Pipeline.Workspace)/packages-signed/**/*.nupkg;!$(Pipeline.Workspace)/packages-signed/**/*.symbols.nupkg'
-          publishVstsFeed: $(DevOpsFeedID)
-      - task: MSBuild@1
-        displayName: 'Publish to blobfeed'
-        condition: and(succeeded(), eq('${{ parameters.DevOpsFeedID }}', variables['DevOpsFeedID']))
-        inputs:
-          solution: '$(AzureSDKBuildToolsPath)/tools/blobfeedtool/BlobFeedPublishHelper.proj'
-          msbuildArguments: '/p:AccountKey=$(azuresdkartifacts-access-key) /p:ExpectedFeedUrl=$(BlobFeedUrl) /p:PackagesPath="$(Pipeline.Workspace)/packages-signed"'
```

Comment:
Can you also look into removing this keyvault secret from our pipelines and maybe even from our keyvault.

Reply:
I Removed this from the pipeline and KeyVault